scripts/docs/bootstrap_api.py

```diff
@@ -12,6 +12,7 @@
 
 def get_signature(cmd):
     source = inspect.getsource(cmd)
+    source = "\n".join(l for l in source.split("\n") if not l.startswith("@"))
```

Comment:
We should probably document this script, at least internally with docstrings or in https://mlem.ai/doc/contributing/core.